libraries/AP_Common/AP_Common.h

```diff
@@ -86,6 +86,7 @@
 #define ARRAY_SIZE(_arr) (sizeof(_arr) / sizeof(_arr[0]))
 
 #define UINT16_VALUE(hbyte, lbyte) (static_cast<uint16_t>((hbyte<<8)|lbyte))
+#define UINT32_VALUE(b3, b2, b1, b0) (static_cast<uint32_t>((b3<<23)|(b2<<16)|(b1<<8)|b0))
```

Comment:
please put parentheses around macro values
so (b3)

Reply:
I've added the parentheses to both UINT16_VALUE and UINT32_VALUE and re-tested and it seems to still work.  I'll merge after it passes travis.  Thanks!